zerver/tests/test_signup.py

```diff
@@ -140,6 +140,10 @@ def test_add_new_user_history_race(self) -> None:
         streams = Stream.objects.filter(id__in=[sub.recipient.type_id for sub in subs])
         self.send_stream_message(self.example_email('hamlet'), streams[0].name, "test")
         add_new_user_history(user_profile, streams)
+        last_n_messages = 80
+        unread_messages = UserMessage.objects.filter(user_profile=user_profile).order_by('-id')[:last_n_messages]
+        for message in unread_messages:
```

Comment:
Maybe just test for 80th message from end and a few others (and not all 80 messages) ? Not sure though.

Reply:
changed the test to check the last 6 messages only.